spec.html

```diff
@@ -10697,6 +10706,7 @@ <h2>Syntax</h2>
       SingleLineCommentChar ::
         SourceCharacter but not LineTerminator
     </emu-grammar>
+    <p>A conforming implementation, when parsing source code using the goal symbol |Module|, must not extend, as described in <emu-xref href="#sec-html-like-comments"></emu-xref></p>
```

Comment:
must not extend what?

Reply:
Extend the syntax and semantics. Or should I not mention that and simply add `The semantics and syntax are extended in emu-xref href="#sec-html-like-comments">`